cozy/main.py

```diff
@@ -33,7 +33,9 @@ def run():
     parser = argparse.ArgumentParser(description='Data structure synthesizer.')
     parser.add_argument("-S", "--save", metavar="FILE", type=str, default=None, help="Save synthesis output")
     parser.add_argument("-R", "--resume", action="store_true", help="Resume from saved synthesis output")
-    parser.add_argument("-t", "--timeout", metavar="N", type=float, default=60, help="Per-query synthesis timeout (in seconds); default=60")
+    parser.add_argument("-t", "--timeout", metavar="N", type=float, default=60,
+                        help="Per-query synthesis timeout (in seconds); default=60. " +
+                             "Smaller timeout speeds up the synthesis process, but may also result in less efficient code")
```

Comment:
Ack, it's another case of fixing up already-broken documentation.

In reality, the timeout is global (not per-query). Cozy runs synthesis for that many seconds total.